batch-remote-partition/src/test/java/org/springframework/cloud/dataflow/acceptence/tests/batch/remote/partition/BatchRemotePartitionApplicationTests.java

```diff
@@ -63,8 +63,12 @@ void partitionedJobRuns() throws InterruptedException, ExecutionException, Timeo
 		ExecutorService executor = Executors.newSingleThreadExecutor();
 
 		Future<?> result = executor.submit(() -> {
+			String[] args = {
+					"--logging.level.org.springframework.cloud.deployer.local=DEBUG"
```

Comment:
Do we still need this debug statement?

Reply:
Probably not. This test is disabled anyway. It's really for use during development, but can't be automated because the deployer uploads the previously built snapshot.